TESTS/configs/baremetal.json

```diff
@@ -35,7 +35,27 @@
         "qspif"
     ],
     "target_overrides": {
-        "*": {
+        "K64F": {
+            "target.default_lib": "small",
+            "mbed-trace.fea-ipv6": false
```

Comment:
What is that and why does it need to be set?

Reply:
I'll let rajkumar answer. But `mbed-trace.fea-ipv6` should have been overridden for all targets.

Reply:
"mbed-trace.fea-ipv6": false added to disable IPV6 tracing code in mbed_trace and we need mbed_trace due to dependency with the device key. More information on PR https://github.com/ARMmbed/mbed-os/pull/11875